tonic-build/src/server.rs

```diff
@@ -271,22 +277,58 @@ fn generate_trait_methods<T: Service>(
                 generate_doc_comments(method.comment())
             };
 
-        let method = match (method.client_streaming(), method.server_streaming()) {
-            (false, false) => {
+        let method = match (method.client_streaming(), method.server_streaming(), default_impl) {
+            (false, false, true) => {
+                quote! {
+                    #method_doc
+                    async fn #name(&self, request: tonic::Request<#req_message>)
+                        -> std::result::Result<tonic::Response<#res_message>, tonic::Status> {
+                        Err(tonic::Status::unimplemented("Not yet implemented"))
```

Comment:
It would be nice to have this overridable. I proposed something in an issue (i incorrectly raised in the wrong repo) over here: https://github.com/tokio-rs/prost/issues/844#issuecomment-1510479430

Reply:
I have raised a PR to bring it in here: https://github.com/xanather/tonic/pull/1